src/VirtoCommerce.Platform.Web/wwwroot/js/app/security/blades/account-detail.js

```diff
@@ -44,6 +45,12 @@ angular.module('platformWebApp').controller('platformWebApp.accountDetailControl
             return !angular.equals(blade.currentEntity, blade.origEntity) && blade.hasUpdatePermission();
         }
 
+        blade.sendLink = function () {
+            if (blade.isLinkSent === false) {
```

Comment:
1. simplify to !blade.isLinkSent
2. set isLoading = true and lock UI from clicking second time